packages/0x.js/src/order_watcher/event_watcher.ts

```diff
@@ -51,6 +51,10 @@ export class EventWatcher {
     }
     private async _pollForBlockchainEventsAsync(callback: EventWatcherCallback): Promise<void> {
         const pendingEvents = await this._getEventsAsync();
+        if (_.isUndefined(pendingEvents)) {
+            // HACK: This should never happen, but happens frequently on CI due to a ganache bug
```

Comment:
Hm, could we check if we are running in CI by checking an environment variable in this conditional block as well?  That would make it more explicit that this conditional only exists for CircleCI.

Reply:
I'm not sure if it does and why this happens. Personally I'd rather have a warning here, but 0x.js didn't emit any output before and I'm not sure this is a good time to start.
This is smth that should never happen, but it isn't critical and we can recover from that.

Reply:
Ok, then let's just move the `HACK` comment above the conditional and also link to the specific Ganache issue on Github.

Reply:
There is no specific ganache issue because I haven't investigated the cause yet.
I don't feel like reporting smth like: "Your thing returns undefined in random cases" is good enough to create an issue.